packages/mobile/e2e/src/usecases/NewAccountPhoneVerification.js

```diff
@@ -59,6 +52,13 @@ export default NewAccountPhoneVerification = () => {
 
   // Check that Twilio SID, Auth Token and Verification Phone Number are defined
   if (TWILIO_ACCOUNT_SID && TWILIO_AUTH_TOKEN && VERIFICATION_PHONE_NUMBER) {
+    // Log Twilio balance at start
```

Comment:
@MuckT @jeanregisser i moved the twilio balance check into this conditional, to avoid the failure we saw yesterday. hope this is sufficient for now...